pkgs/development/python-modules/torchaudio/default.nix

```diff
@@ -31,27 +34,42 @@ buildPythonPackage rec {
     TORCH_CUDA_ARCH_LIST = "${lib.concatStringsSep ";" torch.cudaCapabilities}";
   };
 
+  # https://github.com/pytorch/audio/blob/v2.1.0/docs/source/build.linux.rst#optional-build-torchaudio-with-a-custom-built-ffmpeg
+  FFMPEG_ROOT = symlinkJoin {
+    name = "ffmpeg";
+    paths = [
+      ffmpeg-full.bin
+      ffmpeg-full.dev
+      ffmpeg-full.lib
+    ];
+  };
+
   nativeBuildInputs = [
     cmake
     pkg-config
     ninja
-  ] ++ lib.optionals cudaSupport [
-    cudaPackages.cuda_nvcc
-  ];
+  ] ++ lib.optionals cudaSupport (with cudaPackages; [
+    cuda_nvcc
+  ]);
+
   buildInputs = [
+    ffmpeg-full
     pybind11
-  ] ++ lib.optionals cudaSupport [
-    cudaPackages.libcurand.dev
-    cudaPackages.libcurand.lib
-    cudaPackages.cuda_cudart # cuda_runtime.h and libraries
-    cudaPackages.cuda_cccl.dev # <thrust/*>
-    cudaPackages.cuda_nvtx.dev
-    cudaPackages.cuda_nvtx.lib # -llibNVToolsExt
-    cudaPackages.libcublas.dev
-    cudaPackages.libcublas.lib
-    cudaPackages.libcufft.dev
-    cudaPackages.libcufft.lib
-  ];
+    sox
+  ] ++ lib.optionals cudaSupport (with cudaPackages; [
+    libcurand.dev
+    libcurand.lib
+    cuda_cudart # cuda_runtime.h and libraries
+    cuda_cccl.dev # <thrust/*>
+    cuda_nvtx.dev
+    cuda_nvtx.lib # -llibNVToolsExt
+    cudatoolkit
```

Comment:
same comment re `cudatoolkit`

Reply:
Huh, so were all of the removed outputs unnecessary?